src/main/resources/english.yml

```diff
@@ -264,3 +264,5 @@ msg_swa_set_wins_success: '&bSuccessfully set wins to %d for %s.'
 msg_swa_set_losses_success: '&bSuccessfully set losses to %d for %s.'
 msg_swa_set_nationdefeats_success: '&bSuccessfully set nation defeats to %d for %s.'
 msg_err_nation_not_registered: '&cNation %s is not registered.'
+

```

Comment:
This will probably need the lang file version bumped if it hasn't been bumped since the 0.1.0 release.

Reply:
Lang file was bumped in #95